examples/moodbot/Makefile

```
@@ -39,7 +39,7 @@ run-webexteams:
 	python -m rasa_core.run -d models/dialogue -u models/nlu/current -p 5002 -c webexteams --credentials credentials.yml
 
 run-cmdline:
-	python -m rasa_core.run -d models/dialogue -u models/nlu/current
+	python -m rasa_core.run -d models/dialogue -u models/nlu/current --debug
```

probably should be removed again
```suggestion
	python -m rasa_core.run -d models/dialogue -u models/nlu/current
```